### ISSUE TITLE
IPFS Cannot connect to https resources on IPFS because of insecure connection

### ISSUE THREAD

**Issue by Mithrandir2k18** [brave/brave-browser#13853]
## Description 
Some applications like https://orbit.chat/ require an https connection to the ipfs node.
The node that comes with brave apperently doesn't support that, as I get 
![image](https://user-images.githubusercontent.com/41417290/106371028-25f8ba80-6360-11eb-815e-4dc7ffa625af.png)

once the address is resolved to https://orbit.chat.ipns.localhost:9006/.

Now the annoying thing is, I cannot even connect to an https IPFS gateway, because it'll automatically override the address to point to the local node, that doesn't support https. I tried the address on chrome and there both, going through the https gateway https://ipfs.io/ipfs/QmU9ZTANSMd78GhdDv7srSBeiFkffSFqADpEJn3E4X7Thw/
and the above orbit.chat address work.

So basically with applications like orbit, they will work if you use a browser that doesn't support IPFS.

## Steps to Reproduce
   1. Install the IPFs extention and go to https://orbit.chat/. It won't work.
   2. Use e.g. google chome and https://orbit.chat/ will work

**Comment by diracdeltas** [comment]
@lidel any suggestion on how we should handle this?

**Comment by lidel** [comment]
@Mithrandir2k18 mind providing more information?

1. where does `localhost:9006` come from? (this is not the default port of Brave node)
2. Are you running IPFS Companion? 
    2.1 What _IPFS Node Type_ is set in its _Preferences_?
    2.2 What is IPFS API URL in its _Preferences_?

**Comment by lidel** [comment]
Ok, turns out this is a bug in https://orbit.chat/  – I filled https://github.com/orbitdb/orbit-web/issues/155

@diracdeltas nothing to fix in Brave nor Companion, this can be closed.

**Comment by diracdeltas** [comment]
@lidel thank you for looking into this!
